src/iop/colorin.c

```diff
@@ -1764,6 +1764,89 @@ void reload_defaults(dt_iop_module_t *module)
+                                                    user_profile_dir, system_profile_dir);
+  g_free(system_profile_dir);
+  g_free(user_profile_dir);
+
+  // In case of embedded ICC profile we will modify tooltip with profile info,
+  // otherwise reset tooltip to generic text.
+  if((color_profile == DT_COLORSPACE_EMBEDDED_ICC) && g)
+  {
+    cmsHPROFILE cmsprofile = cmsOpenProfileFromMem(img->profile, img->profile_size);
+
+    char iccDesc[64]; iccDesc[0] = '\0';
+    cmsGetProfileInfoASCII(cmsprofile, cmsInfoDescription, "en", "US", iccDesc, 64);
+    char iccManuf[64]; iccManuf[0] = '\0';
+    cmsGetProfileInfoASCII(cmsprofile, cmsInfoManufacturer, "en", "US", iccManuf, 64);
+    char iccModel[64]; iccModel[0] = '\0';
+    cmsGetProfileInfoASCII(cmsprofile, cmsInfoModel, "en", "US", iccModel, 64);
+
+    // This is the only profile field in which, although it usually contains a short
+    // copyright text, in theory profile creators can put as long text as they want. So,
+    // we can't take a fast approach of statically allocating memory of some sufficient size
+    // and have to find out the size of the field at run-time and dynamically allocate memory.
+    char* iccCopyr;
+    guint32 bufsize = cmsGetProfileInfoASCII(cmsprofile, cmsInfoCopyright, "en", "US", NULL, 0);
+    if(bufsize)
+    {
+      iccCopyr = malloc(bufsize+1);
+      cmsGetProfileInfoASCII(cmsprofile, cmsInfoCopyright, "en", "US", iccCopyr, bufsize);
+    }
+    else
+    {
+      iccCopyr = "";
+    }
+
+    cmsFloat64Number iccVersion = cmsGetProfileVersion(cmsprofile);
+    char *iccType = "";
+    if(cmsIsMatrixShaper(cmsprofile)) iccType = "Matrix";
+    else if(cmsIsCLUT(cmsprofile, INTENT_PERCEPTUAL, LCMS_USED_AS_INPUT)) iccType = "LUT";
+
+    char *tooltip = g_strdup_printf(_("embedded ICC profile properties:\n\n"
+                                    "name: <b>%s</b>\n"
+                                    "version: <b>%g</b>\n"
+                                    "type: <b>%s</b>\n"
+                                    "manufacturer: <b>%s</b>\n"
+                                    "model: <b>%s</b>\n"
+                                    "copyright: <b>%s</b>\n\n%s"),
+                                    iccDesc,
+                                    iccVersion,
+                                    iccType,
+                                    iccManuf,
+                                    iccModel,
+                                    iccCopyr,
+                                    tooltip_part_profile_dirs);
+    gtk_widget_set_tooltip_markup(g->profile_combobox, tooltip);
+    g_free(tooltip);
+    g_free(tooltip_part_profile_dirs);
+    if(bufsize) free(iccCopyr);
+  }
+  else
+  {
+    // If the current image does not have an embedded profile, let's display a generic tooltip
+    gtk_widget_set_tooltip_text(g->profile_combobox, tooltip_part_profile_dirs);
+    g_free(tooltip_part_profile_dirs);
```

Comment:
Ditto, could just say along the lines of a fixed string "no embedded ICC profile"...

Reply:
I haven't caught your idea yet. Are you insisting that the user should learn about folders for external input ICC profiles _only_ from the documentation?

Reply:
No. I think I remember seeing such a tooltip already somewhere, so that's why I brought it up. I'll double-check...

So the idea is just to avoid clutter and duplication, if that is the case of course.

Reply:
Here you go:

https://github.com/darktable-org/darktable/blob/09d99ce81f139ee9c7dd60f8c73ad6f35503861d/src/iop/colorin.c#L1958-L1967

Reply:
Maybe there is a way to present and refactor this "better" somehow, but for now I don't see a need to duplicate it.

Reply:
> There is already a tooltip with this same exact information for both "input profile" and "working profile".

See above

> Maybe there is a way to present and refactor this "better" somehow, but for now I don't see a need to duplicate it.

See above